substrate/runtime/src/lib.rs

```diff
@@ -388,9 +502,12 @@ construct_runtime!(
     TransactionPayment: transaction_payment,
 
     Assets: assets,
+    PoolAssets: assets::<Instance1>::{Pallet, Call, Storage, Event<T>},
```

Comment:
Actually probably the best way to do this now that I think about it :+1: Congrats for figuring this out.

Prob best to rename it `LiquidityTokens` though? The current one is ambiguous about tokens in-pool vs tokens of pool.